yb-voyager/src/callhome/diagnostics.go

```diff
@@ -255,11 +255,10 @@ func MarshalledJsonString[T any](value T) string {
 
 // [For development] Read ENV VARS for value of SendDiagnostics
 func ReadEnvSendDiagnostics() {
-	rawSendDiag := os.Getenv("YB_VOYAGER_SEND_DIAGNOSTICS")
-	for _, val := range []string{"0", "no", "false"} {
```

Comment:
any reason removing the other accepted values `0`, `no` for this ENV?

Reply:
`utils.GetEnvAsBool()` already covers that.
It internally check multiple possible values for both true and false case.

Reply:
oh okay.